docs/reasons.rst

```diff
@@ -45,16 +45,16 @@ Standing on the shoulder of giants
     PyScaffold incentives its users to use the best tools and practices
     available in the Python ecosystem.
 
-    A generated project will contain solid configurations for
-    setuptools_ (the de facto standard for Python packages), Sphinx_ (the de
-    facto Python documentation tool), pytest_ (the de facto Python test runner)
-    and tox_ (the de facto tool for managing Python developments), so the users
-    can run common tasks as easily as typing ``tox -e build``, ``tox -e docs``,
-    or ``tox -e publish`` on the command line.
+    A generated project will contain sane default configurations for
+    setuptools_ (the de facto standard for building Python packages), Sphinx_ (the one & only
+    Python documentation tool), pytest_ and tox_ (most commonly used Python testing framework & runner),
```

Comment:
Maybe "task runner" instead of just runner? Tox is always difficult to define...